worlds/lingo/__init__.py

```diff
@@ -68,6 +68,30 @@ def generate_early(self):
     def create_regions(self):
         create_regions(self)
 
+        if not self.options.shuffle_postgame:
+            state = CollectionState(self.multiworld)
+            state.collect(LingoItem("Prevent Victory", ItemClassification.progression, None, self.player), True)
+
+            for item in self.player_logic.real_items:
+                state.collect(self.create_item(item), True)
+
+            state.sweep_for_events()
```

Comment:
Okay so this is *probably* minor in the grand scheme of things, but you should probably pass world.multiworld.get_locations(player) here as an argument for performance / scope reasons

In fact, this is completely free since you call that later anyway, so you could just save it to a variable first, then pass it both to this call to `sweep_for_events` and to your `unreachable_locations` list comprehension

Reply:
Done.